### ISSUE TITLE
Initial release?

### ISSUE THREAD

**Issue by tammojan** [lofar-astron/DP3#40]
How about making an initial release of the stand-alone distribution so that packaging can be tested (see #39)?

**Comment by aroffringa** [comment]
Yeah that would be useful, but it might be good to solve #27 first, which requires some testing of pull request #37 .

**Comment by gijzelaerr** [comment]
please use a version that is higher than 3.1.7 to make sure automated updating tools work properly.

**Comment by aroffringa** [comment]
The current version seems mature :). Release a version now?

**Comment by tammojan** [comment]
Yes please!

**Comment by revoltek** [comment]
can we have a quick look at #81 before? that's a quite recurring problem and I think we can pin it down relatively quickly as it's very obvious

**Comment by aroffringa** [comment]
(reposting image here to keep it together with the bugreport -- @revoltek you can paste images in the ticket box :))
![image](https://user-images.githubusercontent.com/11645549/48670398-a7fdbf80-eb17-11e8-8c1a-d39cc2435248.png)
